### ISSUE TITLE
$.browser property is not supported in jQuery

### ISSUE THREAD

**Issue by peremenov** [purtuga/jquery.transform#2]
This property was completely removed in jQuery 1.9 so `$.transform(...)` causing error. You need another way to detect browser and its version.

**Comment by purtuga** [comment]
Yeah, the original author of this library told me that it needed several updates in order to support newer versions of IE, so this is probably not the only one.... 
I have not used this plugin in a long time and am not sure when I will be able to try and fix this... any chance you might want to contribute the changes?

**Comment by peremenov** [comment]
I'll try to find some time at weekend

**Comment by purtuga** [comment]
Thank you..
